docs/competition/teams/process-docs/sr2019-orientation-and-pit-rules.md

```diff
@@ -0,0 +1,81 @@
+## Welcome to Student Robotics 2019!
+
+## Pit Rules and Guidance
+
+The following rules must be observed at all times in the pits:
+
+1.	No battery charging. You must hand over your battery charger and spare batteries at reception. During the competition, visit Helpdesk (on the top floor, see map) to swap a discharged battery for a charged one.
+2.	No powered tools outside of the power tools area on the bottom floor (such as drills, soldering irons, etc.). **Power tools should only be used in the downstairs area.**
+3.	Keep your robot in your pit when possible – we may need to find you quickly.
+4.	Please avoid blocking or restricting aisles through pits.
```

Comment:
Maybe move this rule up so it's grouped with the other safety rules?

Would it make sense to explicitly have separate subsections for safety rules rather than other rules? Just a thought.

Reply:
> Would it make sense to explicitly have separate subsections for safety rules rather than other rules?

The only thing I have against this is I don't want to give them the idea that any rules are more important than any of the others